.travis.yml

```diff
@@ -55,11 +58,11 @@ before_script:
   - cp habitica.resources.example habitica.resources
   - cp Habitica/google-services.json.example Habitica/google-services.json
 script:
-  - ./gradlew $TEST
+  - ./gradlew $TEST1
```

Comment:
If we can currently only run one of the tests, I would prefer to just run the `testProdDebugUnitTest` one. Since the `assembleDebug` does not actually run any tests but just confirms that the app can build a valid apk. (which is also useful, but having tests running is a bit more useful I think)